src/main/resources/templates/animal/list.ftl

```diff
@@ -1,16 +1,438 @@
-<#include "../header.ftl">
-<#assign pageSubmissionUrl = animalStatus???then('&animalStatus=${animalStatus}','') >
-<div class="container">
+<!DOCTYPE html>
+<#assign sec=JspTaglibs["http://www.springframework.org/security/tags"]/>
+<html lang="en-US"
+      class=" js flexbox canvas canvastext webgl no-touch geolocation postmessage websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers no-applicationcache svg inlinesvg smil svgclippaths js_active  vc_desktop  vc_transform  vc_transform "
+      style="height: auto; overflow: auto;">
```

Comment:
There is no need to rewrite  the list page. Revert these changes!

Reply:
Cnages reverted